.circleci/config.yml

```diff
@@ -342,6 +342,7 @@ jobs:
       - NODE_VERSION: << pipeline.parameters.node-version >>
     steps:
       - build-win
+      - package-extensions
```

Comment:
I had to update how we run the 'test:ui' script as it was building a vsix file on the fly previously and now we need to following the bundling path.